src/Mod/Draft/draftguitools/gui_rotate.py

```diff
@@ -235,19 +234,26 @@ def set_rotation_angle(self, arg):
 
     def set_ghosts(self):
         """Set the ghost to display."""
+        for ghost in self.ghosts:
+            ghost.remove()
         if self.ui.isSubelementMode.isChecked():
-            return self.set_subelement_ghosts()
-        self.ghosts = [trackers.ghostTracker(self.selected_objects)]
+            self.ghosts = self.get_subelement_ghosts()
+        else:
+            self.ghosts = [trackers.ghostTracker(self.selected_objects)]
+        if self.center:
+            for ghost in self.ghosts:
+                ghost.center(self.center)
 
-    def set_subelement_ghosts(self):
-        """Set ghost for the subelements (vertices, edges)."""
+    def get_subelement_ghosts(self):
```

Comment:
You called other functions setSubelementMode... then maybe this one should also be without underscores? Same goes for the whole PR?
```suggestion
    def getSubelementGhosts(self):
```

Reply:
I have used `setSubelementMode` in `DraftGui.py` because similar functions in that file use that naming scheme. The name `get_subelement_ghosts` complies with what is the new naming scheme for the Draft WB.